modules/test/playwright/tests/portal-search-web/searchFacets.spec.ts

```diff
@@ -52,6 +52,16 @@ test.describe('Clear and retain facet selections', () => {
 		await searchPage.selectSearchFacetLink(lastModifiedPastYearFacetLink);
 	});
 
+	test.afterEach(async ({searchPage}) => {
+
+		// Teardown by resetting search options
+
+		await searchPage.selectSearchOptionConfigurations(
+			['Allow Empty Searches', 'Retain Facet Selections Across Searches'],
+			false
+		);
```

Comment:
This doesn't need to be made now, but for future reference we can improve this method by passing in an object instead if we want to set different checkboxes true/false. 

_However_, we should rename this method since it since it only handles checkboxes.

```
await searchPage.selectSearchOptionCheckboxConfigurations(
[
  {
    label: 'Allow Empty Searches',
    value: false
  },
  {
    label: 'Retain Facet Selections Across Searches',
    value: false
  }
]
);
```

Reply:
Ok! I ended up making the change with the object array parameter as well, since yes it makes sense for varying the configurations in the future.